scip-java/src/main/scala/com/sourcegraph/scip_java/buildtools/GradleBuildTool.scala

```diff
@@ -40,7 +40,7 @@ class GradleBuildTool(index: IndexCommand) extends BuildTool("Gradle", index) {
           "gradlew"
       )
     val gradleCommand: String =
-      if (Files.isRegularFile(gradleWrapper))
+      if (Files.isExecutable(gradleWrapper))
```

Comment:
Do we wanna check both? 
> on UNIX systems, checking for execute access checks that the Java virtual machine has permission to search the directory in order to access file or subdirectories.
https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#isExecutable-java.nio.file.Path-

Reply:
This would return true for directories too, or were we only using isRegularFile to test for existence,m

Reply:
Directories return false for me 🤔 
```
@ java.nio.file.Files.isExecutable(java.nio.file.Paths.get("project"))
res0: Boolean = false
```

Reply:
Interesting, contradicts the javadoc 🤔 id assume macos falls under the Unix umbrella

Reply:
It doesn't hurt to test both 🤷🏻 updated the PR